src/amuse/units/nist.py

```diff
@@ -45,157 +50,213 @@ def get_table_from_url(self):
         f.close()
 
     def save_table_as(self, filename):
-        f = open(os.path.join(self.directory, 'nist.txt'), 'w')
+        f = open(os.path.join(self.directory, self.nist_filename), "w")
         f.write(self.nist_table)
         f.close()
-     
+
     def get_table_from_file(self):
-        f = open(os.path.join(self.directory, 'nist.txt'), 'r') # CODATA 2006, for CODATA 2010 use 'nist2010.txt'
-        self.nist_table = f.read()                             
-        f.close()                                    
+        f = open(
+            os.path.join(self.directory, self.nist_filename), "r"
+        )  # CODATA 2006, for CODATA 2010 use 'nist2010.txt'
+        self.nist_table = f.read()
+        f.close()
 
     def check_current_file_with_table(self):
-        md5sum_local = md5()                         
-        md5sum_local.update(self.local_table)              
-        md5sum_local_val = md5sum_local.hexdigest()  
-        md5sum_wgot = md5()                          
-        md5sum_wgot.update(self.nist_table)          
-        md5sum_wgot_val = md5sum_wgot.hexdigest()    
-        return md5sum_local_val == md5sum_wgot_val   
+        md5sum_local = md5()
+        md5sum_local.update(self.local_table)
+        md5sum_local_val = md5sum_local.hexdigest()
+        md5sum_wgot = md5()
+        md5sum_wgot.update(self.nist_table)
+        md5sum_wgot_val = md5sum_wgot.hexdigest()
+        return md5sum_local_val == md5sum_wgot_val
 
     def compare_char_by_char(self):
 
-        self.nist_table.lstrip('\n')
-        mydiff = difflib.unified_diff(self.nist_table.splitlines(1), self.local_table.splitlines(1))
+        self.nist_table.lstrip("\n")
+        mydiff = difflib.unified_diff(
+            self.nist_table.splitlines(1), self.local_table.splitlines(1)
+        )
         for i in list(mydiff):
             print(i)
-            
+
     def get_translator(self):
-        f = open(os.path.join(self.directory, 'translator.txt'), 'r')     
+        f = open(os.path.join(self.directory, "translator.txt"), "r")
```

Comment:
Please keep real changes and formatting changes from black in separate commits, can't see the forest for the trees here...